image/templates/helm/shared/templates/_metadata.tpl

```diff
@@ -169,7 +169,13 @@
   {{- $name -}}
 {{- else -}}
   {{- /* Add global prefix to resource name. */ -}}
-  {{- printf "%s-%s" $._rox.globalPrefix (trimPrefix "stackrox-" $name) -}}
+  {{- if hasPrefix "stackrox-" $name -}}
+    {{- printf "%s-%s" $._rox.globalPrefix (trimPrefix "stackrox-" $name) -}}
+  {{- else if hasPrefix "stackrox:" $name -}}
+    {{- printf "%s:%s" $._rox.globalPrefix (trimPrefix "stackrox:" $name) -}}
```

Comment:
I have added a new prefixing scheme to keep all the global resources named `stackrox:xxx` unchanged.